exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,9 @@
 // Start by creating a variable `message`
-
-console.log(message);
+// var message;
+// console.log(message);
+// var message="hello world. I just started learning JavaScript!."
+// console.log(message);
+var myName="Luna";
+var nameLength=myName.length;
+var message= "my name is " + myName + " my name is " + nameLength +" characters long."; 
```

Comment:
Hi Luna, in line 8 you can also do "my name is ${myName} + my name is ${nameLength} + characters long.";